spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -133,7 +133,7 @@ object SpdxLicenseAliasMapping {
     /**
      * The map of deprecated SPDX license names associated with their current SPDX expression.
      */
-    private val deprecatedLicenses = mapOf(
+    private val deprecatedLicenseNames = mapOf(
```

Comment:
The correct name per SPDX spec for the variable should be `deprecatedLicenseIds`as this is a map of SPDX license identifiers. See also column names in table on https://spdx.org/licenses/ name is "BSD Zero Clause License" and identifier is 0BSD

Reply:
Makes sense, but then I'll also rename the existing `customNames` to `customIds`, or yet better `customLicenseIds`.

Reply:
Prefer the later `customLicenseIds`